packages/cozy-harvest-lib/src/index.js

```diff
@@ -19,6 +20,10 @@ export const TriggerManager = (props, context) => (
   </I18n>
 )
 
+TriggerManager.contextTypes = {
+  lang: PropTypes.string.isRequired
+}
+
```

Comment:
Not for AccountForm as well?

Reply:
Actually it is not used anymore, and should not. I'll remove it from being exposed here.